deploy/static/install.sh

```diff
@@ -32,6 +32,18 @@ install_deb() {
     apt-get install -y husarnet
 }
 
+install_pacman() {
+    wget -q https://install.husarnet.com/repo.key -O /tmp/repo.key
+    pacman-key --add /tmp/repo.key >/dev/null 2>/dev/null
+    pacman-key --lsign-key 197D62F68A4C7BD6 >/dev/null 2>/dev/null
+    rm /tmp/repo.key
+    echo '[husarnet]' >> /etc/pacman.conf
```

Comment:
It'd be nice if you made this so users can run this script multiple times on the same system (as they seem to be doing), without cluttering the file with multiple entries.